pkg/rebuild/debian/strategy.go

```diff
@@ -81,13 +81,24 @@ func (b *Debrebuild) ToWorkflow() *rebuild.WorkflowStrategy {
 				"buildinfoMd5": b.BuildInfo.MD5,
 			},
 		}},
-		Build: []flow.Step{{
-			Uses: "debian/build/debrebuild",
+		Deps: []flow.Step{{
+			Uses: "debian/deps/install",
 			With: map[string]string{
-				"targetPath": "{{.Target.Artifact}}",
-				"buildinfo":  path.Base(b.BuildInfo.URL),
+				"requirements": flow.MustToJSON([]string{"devscripts", "apt-utils", "mmdebstrap"}),
 			},
 		}},
+		Build: []flow.Step{
+			{
+				Uses: "debian/patch/debrebuild",
```

Comment:
i'd name this debian/deps/patch-debrebuild and move this ref to Deps

the second path element was meant to be a phase grouping and I think in this case the patching is really more setting up the dep than it is part of the build